manifests/profile/hathitrust/php.pp

```diff
@@ -14,31 +14,53 @@
 
   package {
     [
-      'php7.0-cli',
-      'php7.0-common',
       'php7.0-curl',
       'php7.0-gd',
-      'php-geoip',
+      'php-geoip', # PECL
       'php7.0-ldap',
       'php7.0-mysql',
-      'php7.0-xsl',
-      'php-date',
-      'php-db',
-      'php-http-request',
-      'php-log',
-      'php-mail',
       'php-mdb2',
-      'php-net-smtp',
-      'php-net-url2',
-      'php-pear',
+      'php-mdb2-driver-mysql',
+      'php-smarty',
+      'php7.0-xsl',
       'libapache2-mod-php7.0',
       'pear-channels'
     ]:
   }
 
-#      to install via pear
-#      'php-pager',
-#      'php-xml-parser',
-#      'php-xml-serializer',
+  class { '::php':
+    ensure       => present,     # Don't touch stuff from above; should be equivalent
+    manage_repos => false, # Set true to add dotdeb repos
+    fpm          => false,          # We only use mod_php at present
+    composer     => false,     # System-wide composer seems iffy unless using dotdeb
+    pear         => true,          # We're using this for PEAR, so set to true
+    phpunit      => true,       # Unsure whether this should be system or app-level
 
+    extensions   => {
```

Comment:
We probably don't want to specify the ones that come by default with pear, since tiger will complain that the installed files no longer match the package.

Reply:
In particular, that's:

``Archive_Tar
Console-Getopt
PEAR
PEAR_Manpages
Structures_Graph
XML_Util```

Reply:
That sounds to me like a problem with tiger rather than with this setup. Specifying the exact extensions we need seems like a Good Thing to me, so I feel like we shouldn't be optimizing our config management for a system validator that we know gives false positives. But I guess, ask me again after a few weeks of emails.

Reply:
I'm not convinced we actually even _need_ those extensions so much as they're the ones installed by default and we just listed all the ones that were installed on moxie-1.

Reply:
I am going to go ahead and remove them for now.